zhaquirks/xiaomi/aqara/tvoc.py

```diff
@@ -66,6 +67,24 @@ async def bind(self):
         return result
 
 
+class TVOCDisplayUnit(t.enum_factory(t.uint8_t)):
+    """Display values."""
+
+    mgm3_celsius = 0x00
+    ppb_celsius = 0x01
+    mgm3_fahrenheit = 0x10
+    ppb_fahrenheit = 0x11
+
+
+class TVOCCluster(XiaomiAqaraE1Cluster):
+    """Aqara LUMI Config cluster."""
+
+    ep_attribute = "aqara_cluster"
```

Comment:
Generally, `opple_cluster` is used for Aqara devices. This is what ZHA mostly uses to match Aqara devices.
The cluster handler there is also called something with "Opple" at the moment.

However, I think ZHA doesn't yet create a config entity for this. This could be added by doing the following:

- Set `ZCL_INIT_ATTRS` to `display_unit` if the model matches `lumi.airmonitor.acn01` [here](https://github.com/home-assistant/core/blob/9d91cfa27fc10b584114378ee13989e546c03267/homeassistant/components/zha/core/cluster_handlers/manufacturerspecific.py#L111-L116).
- Create a config entity [similar to this here](https://github.com/home-assistant/core/blob/9d91cfa27fc10b584114378ee13989e546c03267/homeassistant/components/zha/select.py#L350-L359).
Note: You'd probably just use `TVOCDisplayUnit`, since `zhaquirks` already defines that (with this PR). Most other quirks do not do that in `zhaquirks`, but directly in ZHA.

If you want (and have some time), feel free to implement that (and PR to HA Core).